CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+
+This is a set of guidelines for contribuiting to [DQ Robotics](https://dqrobotics.github.io/).
+
+
+
+
+# Workflow :nerd_face:
+
+- Fork the [master branch of the dqrobotics/matlab](https://github.com/dqrobotics/matlab).
+- Propose your modifications and open a draft pull request.
+- Your modifications will be tested automatically by Github actions. Specifically, Github actions runs the tests of [matlab-tests](https://github.com/dqrobotics/matlab-tests), which executes all the examples of [matlab-examples](https://github.com/dqrobotics/matlab-examples). 
+- Once your draft pull request passes all the tests, you can switch the status to pull request. (More details [here](https://github.blog/2019-02-14-introducing-draft-pull-requests/)).
+
+![Untitled Diagram](https://user-images.githubusercontent.com/23158313/150255548-b4475747-444e-4530-9b20-def1655bb4f9.png)
+
+## Case 1 (Common cases :hugs:)
+
+- If your modifications pass all tests, Prof. Bruno Adorno and Prof. Murilo Marinho will review all the changes and they will accept them after all necessary adjustments if any. 
+
+## Case 2 (Very rare cases :thinking:)
+- In some cases, your modifications would fail some tests because of incompatibility with the current version of [matlab-tests](https://github.com/dqrobotics/matlab-tests) and/or [matlab-examples](https://github.com/dqrobotics/matlab-examples). In thoses cases, you would propose changes in [matlab-tests](https://github.com/dqrobotics/matlab-tests) and [matlab-examples](https://github.com/dqrobotics/matlab-examples) to make them compatible with your new version of the dqrobotics/matlab. 
+-  Prof. Bruno Adorno and Prof. Murilo Marinho will review the changes proposed in both [matlab-tests](https://github.com/dqrobotics/matlab-tests) and [matlab-examples](https://github.com/dqrobotics/matlab-examples). They will accept the modifications in the master branch after all necessary adjustments. At this point, it is expected that your pull request passes all the tests in the master branch but fails in the release branch.
+
+![master_and_release](https://user-images.githubusercontent.com/23158313/150379489-cabc85bb-dbe4-41be-a405-7b254a36092a.png)
+
+-  Then, they will test your dqrobotics/matlab pull request making all necessary adjustments until your pull request of dqrobotics/matlab passes all the tests. After that, they will review deeply your modifications. 
+-  Finally, Prof. Bruno Adorno and Prof. Murilo Marinho will accept your modifications in the master branch.
+
+
+# Example 	:grin::desktop_computer::keyboard::computer_mouse:
+
+## Fork the [dqrobotics/matlab](https://github.com/dqrobotics/matlab) in your Github account
+
+![fork_master](https://user-images.githubusercontent.com/23158313/149602838-133f6c09-2e16-418e-8ab6-47fb36a91056.gif)
+
+## Clone the forked repository
+
+Type in your terminal:
+
+- `git clone https://github.com/you_user/matlab.git`
```

Comment:
`your_user
`

Reply:
@mmmarinho, modified the explanation about how to clone the forked repository. I removed "your_user".